Joomla/Sniffs/Commenting/ClassCommentSniff.php

```diff
@@ -6,18 +6,17 @@
  * @copyright  Copyright (C) 2015 Open Source Matters, Inc. All rights reserved.
  * @license    http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License Version 2 or Later
  */
+namespace Joomla\Sniffs\Commenting;
 
-if (class_exists('Joomla_Sniffs_Commenting_FileCommentSniff', true) === false)
-{
-	throw new PHP_CodeSniffer_Exception('Class Joomla_Sniffs_Commenting_FileCommentSniff not found');
-}
-
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
 /**
```

Comment:
Interesting. I thought it was something to do with the Travis environment, because it worked fine "in the wild." (As I was unable to get the unit tests up and running in any form, I ran it against Joomla as a test case w/o error -- though with flags.) 

Amended the PR with almost that. I made is as JoomlaFileCommentSniff to keep with the form I used for extending a PEAR sniff in FunctionCommentSniff.